### ISSUE TITLE
Failing tests using `mockito`

### ISSUE THREAD

**Issue by elliette** [flutter/devtools#5843]
Example error:

```
loading /home/runner/work/devtools/devtools/packages/devtools_app/test/debugger/debugger_screen_paused_test.dart (failed after test completion)
  
../devtools_test/lib/src/mocks/generated.mocks.dart:108:8: Error: Error when reading '../../../../../.pub-cache/hosted/pub.dev/mockito-5.4.0/lib/src/dummies.dart': No such file or directory
import 'package:mockito/src/dummies.dart' as _i57;
       ^
../devtools_test/lib/src/mocks/generated.mocks.dart:5086:20: Error: Method not found: 'dummyValueOrNull'.
              _i57.dummyValueOrNull<T>(
                   ^^^^^^^^^^^^^^^^
../devtools_test/lib/src/mocks/generated.mocks.dart:5085:27: Error: Method not found: 'ifNotNull'.
        returnValue: _i57.ifNotNull(
                          ^^^^^^^^^
../devtools_test/lib/src/mocks/generated.mocks.dart:8344:20: Error: Method not found: 'dummyValueOrNull'.
              _i57.dummyValueOrNull<T>(
                   ^^^^^^^^^^^^^^^^
../devtools_test/lib/src/mocks/generated.mocks.dart:[8343](https://github.com/flutter/devtools/actions/runs/5072134571/jobs/9109461370?pr=5842#step:4:8459):27: Error: Method not found: 'ifNotNull'.
        returnValue: _i57.ifNotNull(
                          ^^^^^^^^^
```

**Comment by polina-c** [comment]
@elliette , can you give more details? What is the problem and what are steps to repro?

**Comment by elliette** [comment]
Summarizing discussion from `hackers-devtools`:

- mockito 5.4.1 probably should have been a major bump but wasn't
- we were version solving to mockito 5.4.0 in `devtools_app` and 5.4.1 in `devtools_test`. Therefore we were generating mocks with a mockito version that was incompatible than the one we were testing with
- https://github.com/flutter/devtools/pull/5844 pinned the mockito versions to 5.4.0, because https://github.com/flutter/flutter/issues/127226 was preventing us from updating to 5.4.1 in `devtools_app`
- Now that https://github.com/flutter/flutter/issues/127226 is resolved, we should be able to update both to `^5.4.1`. Will open a PR for that
